tachyon/c/zk/plonk/halo2/bn254_gwc_prover_unittest.cc

```diff
@@ -28,6 +31,14 @@ class GWCProverTest : public testing::TestWithParam<int> {
  public:
   using PCS = GWCExtension<math::bn254::BN254Curve, c::math::kMaxDegree,
                            c::math::kMaxDegree, math::bn254::G1AffinePoint>;
+  using Poly =
+      tachyon::math::UnivariateDensePolynomial<tachyon::math::bn254::Fr,
+                                               c::math::kMaxDegree>;
+  using Evals = tachyon::math::UnivariateEvaluations<tachyon::math::bn254::Fr,
+                                                     c::math::kMaxDegree>;
+  using Commitment = tachyon::math::bn254::G1AffinePoint;
+
+  using LS = tachyon::zk::lookup::halo2::LookupScheme<Poly, Evals, Commitment>;
```

Comment:
```suggestion
  using LS = tachyon::zk::lookup::halo2::LookupScheme<PCS::Poly, PCS::Evals, PCS::Commitment>;
```

Reply:
I think this is related to https://youtu.be/eRKn5uFK0V0?t=439. In order to make sure `Native compatibility`, `LS` should use the same `Poly`, `Evals` and `Commitment`.